.github/workflows/ci.yml

```diff
@@ -367,21 +367,15 @@ jobs:
 
             `;
 
-            // 3. Delete previous comment so PR timeline makes sense
+            // 3. Update previous comment
             if (botComment) {
-              github.rest.issues.deleteComment({
+              github.rest.issues.updateComment({
```

Comment:
We can't update a comment that doesn't exist, so we need to conditionally create it. You can see that there is no bot comments in this PR because of this issue.

Reply:
👍 Update incoming